lib/consensus/raft/config.go

```diff
@@ -97,8 +97,8 @@ func NewClusterRaftConfig(userRaftConfig *config.UserRaftConfig) *ClusterRaftCon
 
 }
 
-//// Validate checks that this configuration has working values,
-//// at least in appearance.
+// // Validate checks that this configuration has working values,
```

Comment:
This is from my PR but could you remove the 2nd set of // if changing this

Reply:
Yeah, some linter started having opinions about this recently, we will probably be seeing a bunch of diffs like this for a while.